src/Test.UnitTests.BinSkim.Driver/CompilerDataLoggerTests.cs

```diff
@@ -25,11 +27,14 @@ namespace Microsoft.CodeAnalysis.BinSkim.Rules
     public class CompilerDataLoggerTests
     {
         private const string SarifPath = @"C:\example.sarif";
+        private const string SampleSarifPath = "Native_x86_VS2019_SDL_Enabled.exe.sarif";
+        private const string ExpectedFolder = "Expected";
+        private const string TargetUriPath = @"c:\file.dll";
 
         [Fact]
         public void CompilerDataLogger_Write_ShouldSendAssemblyReferencesInChunks_WhenTelemetryIsEnabled()
         {
-            var context = new BinaryAnalyzerContext() { TargetUri = new Uri(@"c:\file.dll") };
+            BinaryAnalyzerContext context = CreateTestContext();
```

Comment:
>BinaryAnalyzerContext [](http://example.com/codeflow?start=12&length=21)

dispose all contexts

Reply:
not sure if u are aware, but with the new version of C#, you could do:
```
using var context = new Context();
```
and it will know when to dispose based on ur codeflow.

no `{` or `}` needed.